src/app/onboarding/views/__snapshots__/WelcomeView.spec.ts.snap

```diff
@@ -211,7 +219,7 @@ exports[`WelcomeView.vue renders snapshot 1`] = `
         </div>
         <svg
           class="background svg"
-          longer="false"
+          longer="true"
```

Comment:
**Question**: How come this changed snapshot changed?

Reply:
Because the snapshots are no longer taken in standalone mode